js/constants/appConfig.js

```diff
@@ -151,7 +151,7 @@ module.exports = {
     'general.spellcheck-languages': Immutable.fromJS(['en-US']),
     'search.default-search-engine': 'Google',
     'search.offer-search-suggestions': false, // false by default for privacy reasons
-    'search.use-alternate-private-search-engine': false,
+    'search.use-alternate-private-search-engine': true,
```

Comment:
debatable; alternative (can be done in follow-up) is to make DDG a per-tab setting like Tor